include/grpc/grpc.h

```diff
@@ -237,6 +237,10 @@ GRPCAPI struct census_context *grpc_census_call_get_context(grpc_call *call);
     created for. */
 GRPCAPI char *grpc_channel_get_target(grpc_channel *channel);
 
+/** Request info about the channel. */
```

Comment:
Done.

I've also changed the parameter to be const, to make it clear that it's not modified by the call.